wezterm-gui/src/termwindow/mod.rs

```diff
@@ -3539,6 +3540,16 @@ impl TermWindow {
             }
         }
     }
+
+    fn parse_mouse_cursor_shape(&self, mouse_cursor_shape: &str) -> MouseCursor {
+        match mouse_cursor_shape {
+            "pointer" => MouseCursor::Hand,
+            "text" => MouseCursor::Text,
+            "row-resize" | "ns-resize" => MouseCursor::SizeUpDown,
+            "col-resize" | "ew-resize" => MouseCursor::SizeLeftRight,
+            _ => MouseCursor::Arrow,
+        }
+    }
```

Comment:
Since this doesn't do anything with self, it can just be a "free" function

```suggestion

}

pub fn parse_mouse_cursor_shape(mouse_cursor_shape: &str) -> MouseCursor {
        match mouse_cursor_shape {
            "pointer" => MouseCursor::Hand,
            "text" => MouseCursor::Text,
            "row-resize" | "ns-resize" => MouseCursor::SizeUpDown,
            "col-resize" | "ew-resize" => MouseCursor::SizeLeftRight,
            _ => MouseCursor::Arrow,
        }
}
```

Reply:
I wasn't sure what the preferred approach here was, so thank you for this suggestion! Done in 0cd76fc302db43bc14396130f2fe3b2aec82e500